modules/eco-poison-pill-operator-about.adoc

```diff
@@ -8,6 +8,22 @@
 
 The Poison Pill Operator runs on the cluster nodes and reboots nodes that are identified as unhealthy. The Operator uses the `MachineHealthCheck` controller to detect the health of a node in the cluster. When a node is identified as unhealthy, the `MachineHealthCheck` resource creates the `PoisonPillRemediation` custom resource (CR), which triggers the Poison Pill Operator.
 
+The `SelfNodeRemediation` CR resembles the following YAML file:
+
+[source,yaml]
+----
+apiVersion: self-node-remediation.medik8s.io/v1alpha1
+kind: SelfNodeRemediation
+metadata:
+  name: selfnoderemediation-sample
+  namespace: openshift-operators
+spec:
+status:
+  lastError: <last_error_message> <1>
+----
+
+<1> Displays the last error that occurred during remediation. If no error occurs, the field is left empty.
```

Comment:
Nit: another use case that the error will be empty is a successful remediation which will clear the last error.